core/test/unit/migration_spec.js

```diff
@@ -13,6 +13,11 @@ var should = require('should'), // jshint ignore:line
 
     sandbox = sinon.sandbox.create();
 
+// Clone the fixtures as they get changed during the migration.
+// The initial blog posts will be added a `published_at` property, which
+// would change the fixturesHash.
+fixtures = _.cloneDeep(fixtures);
```

Comment:
This was me idea as well, but unfortunately this way the fixtures hash would still change with every test run. That's why I cloned the fixtures in the test only. I also tested changing the fixtures and the test still works = needs a new fixtures hash.